use simple dialog instead of alertdialog breez_avatar (support for smaller screens)

ubbabeck opened the pull request:
# About pr

In order to support devices with smaller screen and avoid the users profilename not showing we can use `SimpleDialog` instead of `AlertDialog` 


https://github.com/breez/c-breez/assets/36157890/6522f7a2-f155-45a1-a663-22824f9f9669

erdemyerebasmaz commented:
@ubbabeck Definitely an improvement but does this make any visual changes on other devices? We wouldn't want a regression that affects most of our users when we're improving UI for small screen devices.

Nit: Random & Gallery buttons doesn't look good to me, can we add some vertical padding to them?

_Tagging @kingonly to review the UI._

kingonly commented:
I don't understand if this changes the existing UI.
As a rule if thumb, everything should look and work like Breez mobile. If users didn't complain it means we don't need to change anything.

ubbabeck commented:
@kingonly it is not supposed to change the UI, only make it more usable for users with smaller devices and/or using larger font and icon size. The difference if that with this dialog we get more granular control than with the other one.

Thank you for the feedback @erdemyerebasmaz I added some padding in the latest commit [af4d300](https://github.com/breez/c-breez/pull/583/commits/af4d3000b964f4d5a6d6ce2d7febb798f9ad1772) so it looks like breezmobile. 

Here is a photo of how it looks with the default screen size. 
![breez_avatar_normal](https://github.com/breez/c-breez/assets/36157890/1169a256-c717-4532-9383-8403e820714b)

Updated large font and icon size 
![breez_avatar_large](https://github.com/breez/c-breez/assets/36157890/2634180e-a56f-4287-8abb-9e134d1a55bf)

kingonly commented:
If the UI is the same, it's fine by me.
Currently, I have more padding between the text field and the buttons.
![Screenshot_2023-06-21-12-18-55-56_9a3a9060ca72a38f8a1fec1d0c1a640e](https://github.com/breez/c-breez/assets/31890660/83a778c9-8177-4554-8124-a0c53a54c338)

ubbabeck commented:
Updated the padding according to `AlertDialogs` padding setting in the latest rebase [a1ba069](https://github.com/breez/c-breez/pull/583/commits/a1ba069ae6bf0dbeeee8554794268c90ad262f76).
![breez_ava](https://github.com/breez/c-breez/assets/36157890/0a4dc507-d967-491b-bb1b-3eddb02e75ba)

kingonly commented:
Great!

erdemyerebasmaz reviewed (approved):
LGTM
